articles/libraries/auth0-android/passwordless.md

```diff
@@ -0,0 +1,60 @@
+---
+section: libraries
+toc: true
+description: How to use Auth0.Android with passwordless connections
+---
+# Auth0.Android Passwordless Authentication
+
+::: panel-warning Passwordless on Native Platforms
+Passwordless on native platforms is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case. See [Client Grant Types](/clients/client-grant-types) for more information.
+
+Alternatively, you can use Lock Passwordless on Auth0's [Hosted Login Page](/hosted-pages/login).
+:::
+
+Passwordless can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. All of these methods of Passwordless authentication will require two steps - requesting the code, and then inputting the code for verification. 
+
+Note that Passwordless authentication **cannot be used** with the [OIDC Conformant Mode](/oidc-conformant-mode) enabled.
```

Comment:
This URL returns a 404 (I would update it but I'm not sure where you want this to link)

Reply:
Ah, that link is bad in the existing doc. Let's go with `https://auth0.com/docs/api-auth/intro` or something. I'll update it.